binding/unity/Assets/Porcupine/Porcupine.cs

```diff
@@ -444,10 +448,17 @@ private static string GetDefaultModelPath()
 #if !UNITY_EDITOR && UNITY_ANDROID
         public static string ExtractResource(string filePath)
         {
-            string srcPath = Path.Combine(Application.streamingAssetsPath, filePath);
-            string dstPath = Path.Combine(Application.persistentDataPath, filePath);
-            var loadingRequest = UnityWebRequest.Get(srcPath);
+            if (!filePath.StartsWith(Application.streamingAssetsPath))
+            {
+                throw new PorcupineIOException($"File '{filePath}' not found in streaming assets path.");
+            }
+
+            string dstPath = Path.Combine(
+                Application.persistentDataPath,
+                filePath.Remove(0, Application.streamingAssetsPath.Length + 1));
```

Comment:
It would probably be more simple to do a replace operation here. Replace streamingAssetsPath -> persistentDataPath